contracts/src/v0.8/vrf/dev/SubscriptionAPI.sol

```diff
@@ -324,13 +326,14 @@ abstract contract SubscriptionAPI is ConfirmedOwner, IERC677Receiver, IVRFSubscr
   function getActiveSubscriptionIds(
     uint256 startIndex,
     uint256 maxCount
-  ) external view override returns (uint256[] memory) {
+  ) external view override returns (uint256[] memory ids) {
```

Comment:
Just curious: So if you don't use named return variables, there's an additional cost (like an additional temp var copy operation)? What happens in the background if you don't use this approach?

Reply:
I am guessing that there will be an additional temp var copy operation. Not sure, will have to dive into the opcodes generated.